docs/crictl.md

```diff
@@ -30,12 +30,45 @@ debug: true
 ```
 
 ## Download and Inspect a Container Image
+The pull command tells the container runtime to load a container image from a
+container registry.
 ```console
 $ crictl pull busybox
   ...
 $ crictl inspecti busybox
   ... displays information about the image.
 ```
+
+## Directly Load a Container Image
+Another way to load an image into the container runtime is with the load
+command. With the load command you inject a container image into the container
+runtime from a file. First you need to create the container image. For example
```

Comment:
container image tarball
or
container image file?

Reply:
done

Reply:
Not changed? :P